src/ansys/geometry/core/_grpc/_services/v0/designs.py

```diff
@@ -481,3 +489,56 @@ def stream_design_tessellation(self, **kwargs) -> dict:  # noqa: D102
         return {
             "tessellation": tess_map,
         }
+
+    @protect_grpc
+    def assign_midsurface_thickness(self, **kwargs) -> dict:
+        """ "Add a mid-surface thickness to a list of bodies."""
+        # Create the request - assumes all inputs are valid and of the proper type
+        request = AssignMidSurfaceThicknessRequest(
+            bodies_or_faces=kwargs["bodies_or_faces"], thickness=kwargs["thickness"]
+        )
+
+        # Call the gRPC service
+        response = self.commands_stub.AssignMidSurfaceThickness(request)
+
```

Comment:
Even if no output is needed, let's return always an empty dictionary for consistency. Search for other ``return {}`` refs